README.md

```diff
@@ -151,6 +152,28 @@ Here's a yasnippet that makes use of `aya-tab-position`. You need to call
 Comments generated with this will always end in same column position,
 no matter from which indentation level they were invoked from.
 
+## Mixed case templates
+
+You can create mixed case templates setting `aya-case-fold` to `t`. This will result
+in templates where variables that start with a character of a different case will be
+treated as the same variable. The case of the first character will be preserved in the
+resulting snippet.
+
+Using the earlier example with a slight twist:
+
+```
+count_of_~red = get_total("~Red");
+```
+
+Then calling `yas-create`, then `yas-expand`, and finally typing `blue`, the result
```

Comment:
Should be `aya-create` and `aya-expand`. Also, I can't get this use case to work. Using yasnippet 20170310.1724.

Reply:
I've corrected the function names. I'm testing using the same version yasnippet. Quick question, are you updating the variable (`(setq aya-case-fold t)`) prior to calling `aya-create` to create the snippet?

Reply:
Oops, sorry; I've had a long day. Your example does work.